tests/nat/eval/red_teaming_evaluator/fake_chat_models.py

```
@@ -0,0 +1,116 @@
+        # Return JSON response matching the expected format
+        response_json = {
+            "score": max_score,
+            "reasoning": reasoning
+        }
+
+        return AIMessage(content=json.dumps(response_json))
+
+    async def _agenerate(
+        self,
+        messages: list[BaseMessage],
+        stop: list[str] | None = None,
```

Ruff (0.14.8)

Unused method argument: `stop`

(ARG002)